literature.bib

```diff
@@ -19077,6 +19264,15 @@ @book{L09
 	publisher= {"O'Reilly Media, Inc."}
 }
 
+@article{LUV:ICSOFT09,
+	title={{Software Product Line Testing}},
+	author={Lamancha, Beatriz P{\'e}rez and Usaola, Macario Polo and Velthius, Mario Piattini},
+	journal={A Systematic Review. ICSOFT (1)},
+	pages={23--30},
+	year=2009,
+	publisher={Citeseer}
```

Comment:
Do we have a variable for that?

Reply:
No, but Citeseer is only a search platform
The correct publisher would be _Springer_